tests/headers/convert-cpp-comment-to-rust.hpp

```diff
@@ -0,0 +1,16 @@
+#include <stddef.h>
+#include <stdint.h>
```

Comment:
Yeah, tests can't have system includes to pass, since these are machine-dependent.